localstripe/resources.py

```diff
@@ -772,6 +774,9 @@ def __init__(self, name=None, description=None, email=None,
         self.shipping = None
         self.default_source = None
 
+        if payment_method:
```

Comment:
Can you check for `payment_method is not None` here too?

Reply:
Addressed in c62a417